app/assets/javascripts/student_profile/page_container.jsx

```diff
@@ -1,16 +1,15 @@
 import _ from 'lodash';
 import MixpanelUtils from '../helpers/mixpanel_utils.jsx';
+import PropTypes from '../helpers/prop_types.jsx';
+/* import * as ReactHelpers from '../helpers/react_helpers.jsx'; */
```

Comment:
Just a reminder note that we should remove this commented-out line before we merge. 👍